catalog/ui/catalog-ui-search/src/main/resources/OSGI-INF/metatype/org.codice.ddf.catalog.ui.xml

```diff
@@ -188,6 +188,13 @@
             default="true"
             required="false"/>
 
+        <AD id="basicSearchTemporalSelectionDefault"
```

Comment:
Do we expect admins to be configuring this? Could this be an internal configuration instead of exposing it in the UI?

Reply:
As far as I am aware, this was always meant to be admin configurable. It is part of the acceptance criteria, so I assume it should be admin configurable.